log/src/io/pedestal/log.clj

```diff
@@ -333,21 +333,36 @@
 ;; -------------------------
 
 (def ^:dynamic *mdc-context*
-  "This map is copied into the SLF4J MDC.
-  You should interact with this via the `with-context` macro.
+  "This map is copied into the SLF4J MDC when the `with-context` or
+  `with-context-kv` macros are used.  You are free to take control of
+  it for MDC-related purposes as it doesn't directly affect Pedestal's
+  logging implementation.
 
-  This map also includes all options that were passed into `with-context`"
+  This map also includes all options that were passed into `with-context`."
   {})
 
 (def mdc-context-key "io.pedestal")
 
 (defmacro with-context
   "Given a map of keys/values/options and a body,
-  Set the map into the SLF4J MDC via the *mdc-context* binding.
+  Set the map into the MDC via the *mdc-context* binding.
+  The MDC used defaults to SLF4J MDC unless the `io.pedestal.log/mdc`
```

Comment:
Might want to change this to `:io.pedestal.log/mdc` so readers know it's a keyword